ui/lib/core/addon/components/kv-suggestion-input.hbs

```diff
@@ -7,11 +7,10 @@
     <FormFieldLabel for={{this.inputId}} @label={{@label}} @subText={{@subText}} />
   {{/if}}
   <FilterInput
-    id={{this.inputId}}
+    @id={{this.inputId}}
```

Comment:
I don't want to blow out the scope of this PR but it seems like this can be removed (and [here](https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/components/kv-suggestion-input.ts#L57)) since the input self-generates an id